akd/Cargo.toml

```diff
@@ -32,18 +34,19 @@ bincode = { version = "1", optional = true }
 serde = { version = "1", features = ["derive"], optional = true }
 rand = { version = "0.7", optional = true }
 curve25519-dalek = { version = "3.2", optional = true }
-ed25519-dalek = { version = "1", optional = true}
-
-[target.'cfg(all(features = "serde", features = "vrf"))'.dependencies]
-# In the event that VRF's are enabled, AND builder has requested serde support
-# Add the serde flag to the dalek crate
-ed25519-dalek = { version = "1", features = ["serde"]}
+ed25519-dalek = { version = "1", optional = true }
+colored = { version = "2", optional = true }
+once_cell = { version = "1", optional = true }
 
 [dev-dependencies]
 criterion = "0.3"
 serial_test = "0.5"
 proptest = "1.0"
 proptest-derive = "0.3"
+colored = { version = "2" }
+once_cell = { version = "1" }
+
+akd = { path =".", features = ["vrf", "public-tests"] }
```

Comment:
We add ourself back in in the `test` profile in order to adjust the features on the create (i.e. enable the test-logic needed)